fog/overseer/server/src/bin/main.rs

```diff
@@ -41,5 +41,6 @@ fn main() {
             .port(config.overseer_listen_port)
             .unwrap();
 
-    server::initialize_rocket_server(rocket_config, overseer_state);
+    let rocket = server::initialize_rocket_server(rocket_config, overseer_state);
```

Comment:
Consider annotating this helper method with `must_use`, e.g.

```rust
#[must_use = "Use with a Client or call launch"]
fn initialize_rocket_server(...)  {}
```

Reply:
Done.